web-app/Dockerfile.prod

```diff
@@ -0,0 +1,30 @@
+FROM node:12-alpine As development
+
+ENV NODE_ENV=development
+
+WORKDIR /usr/app
+
+COPY package.json yarn.lock ./
+
+RUN yarn
+
+COPY . .
+
+RUN yarn build
+
+FROM node:12-alpine As production
+
+ENV TYPEORM_ENTITIES=dist/**/*.entity.js
+ENV TYPEORM_SYNCHRONIZE=true
+ENV NODE_ENV=production
+
+WORKDIR /usr/app
+
+COPY . .
```

Comment:
I think this line should be useless, you have just to copy your lockfile + your package json

Reply:
Some other files are needed (at least those in public/ and views/) but I still get an error when I copy only these files, so it seems like others are missing. I will create an issue to reduce the production image size and I propose to merge it now.

Reply:
Related issue here : https://github.com/marmelab/Hex/issues/36